source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -0,0 +1,135 @@
+
+## Export as OVA
+Today, users are provided with two ways to migrate VMs and templates between environments. First, one can use an export domain. However, this approach is suitable only for migration between different oVirt environments. Moreover, there is a plan to drop the support for export domains. Second, one can detach a data domain and attach it to another setup. This way is tailored to disaster recovery and less fits the migration flow due to the side effects in detaching a data domain.  
+
+This feature provides an alternative way for a migration between environments by generating an OVA from an existing VM or template. By encapsulating all VM/template data in a single file, we achieve an improved portability. By being aligned with the common OVA specification (rather than our unique structure of a storage domain), we achieve better compatibility with other platforms.  
+
+The OVA file would be streamed into imageio proxy, enabling its creation on a variety of locations (path on host in the data-center, NFS share, etc). In addition, it will include neither snapshots nor the template that the VM may be based on.  
+
+Moreover, this feature enables to export a running VM with no downtime, unlike today that one is required to shutdown the VM in order to be able to export it.
+
+## Enhance Import OVA
+This feature enhances the current support provided for import OVA files accessible to the host by enabling to specify a path to OVA that was generated by oVirt. Import of OVA that was generated by oVirt does not involve the adaptation needed for OVA files generated by other platforms (e.g., installing kvm-specific drivers) and thus, should be faster than importing of such OVA files.  
+
+## Download OVA
+This feature will leverage the implementation of the planned download-image to provide similar functionality for OVA files.  
+
+# Design
+This section details the high-level design of the different aspects described in the previous section.
+
+## Upload OVA
+In principle, uploading an OVA is similar to uploading an image. However, it is more challenging because OVA is a more complex resource than an image.  
+
+The upload process would be implemented differently depending on whether the OVA was generated by oVirt (or more accurately, that the disks within the OVA are of type raw or qcow) or by other source (and thus requires adaptation). In both cases, the user will be able to select a file and then be provided with the VM configuration that allows him to configure the parameters for the import process.  
+
+### Upload OVA generated by oVirt
+The following figure depicts the import process of an OVA that was generated by oVirt:
```

Comment:
Suggested edit:
The figure below shows the OVA import process generated by oVirt: